website/content/docs/agent-and-proxy/proxy/apiproxy.mdx

```diff
@@ -55,6 +55,11 @@ configuration will be overridden and the token in the request will be used to
 forward the request to the Vault server. If set to `"force"` Proxy will use the
 auto-auth token, overwriting the attached Vault token if set.
 
+~> **Note**: When using the proxy's auto-auth token, one proxy per application is very
+   strongly recommended, as Vault will unable to disinguish requests coming from
+   multiple applications through a single proxy. In situations where a single proxy is
+   shared by multiple applications, disabling `use_auto_auth_token` is recommended.
```

Comment:
```suggestion
~> **Note**: When using the proxy's auto-auth token with the `use_auto_auth_token` 
   configuration, one proxy per application is very strongly recommended, as Vault will
   unable to distinguish requests coming from multiple applications through a single proxy. 
   In situations where a single proxy is shared by multiple applications, setting `use_auto_auth_token` 
   to `false` (the default) is recommended.
```

Reply:
One typo (disinguish -> distinguish) and I tried to make it clearer we were talking about this specific config. It's possible to have auto-auth configured on a Proxy that doesn't have the `use_auto_auth_token` setting enabled (e.g. for static secret caching) and that's not necessarily discouraged

Reply:
added in [14b1058](https://github.com/hashicorp/vault/pull/25746/commits/14b1058a928183301577ad3c1b1e3d093d69988e) due to a conflict I introduced shortly after you commented